src/pages/tag.js

```diff
@@ -92,7 +106,12 @@ export default class TagGridPage extends Component {
       <Fragment>
         <Helmet
           title={`${tag && tag.name} | ${shop && shop.name}`}
-          meta={[{ name: "description", content: shop && shop.description }]}
+          meta={
+            tag.metafields && tag.metafields.length > 0 ?
+              this.renderHeaderMetadata(tag.metafields)
+              :
+              [{ name: "description", content: shop && shop.description }]
```

Comment:
If I define metafields for a tag, but omit the `description` field, then I don't get the shop description at all.

We should try to keep the shop's description if one isn't defined for the tag. React Helmet will de-dupe fields, but I'm not sure if it works in the same array; though it's worth trying either way.